src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -99,4 +111,42 @@ private void copyUrl() {
         url.putString(USERGUIDE_URL);
         clipboard.setContent(url);
     }
+
+    /**
+     * Initializes the help window.
+     */
+    @FXML
+    public void initialize() {
+        commandColumn.setCellValueFactory(data -> new SimpleStringProperty(data.getValue()[0]));
+        descriptionColumn.setCellValueFactory(data -> new SimpleStringProperty(data.getValue()[1]));
+
+        commandColumn.setMinWidth(100);
+        descriptionColumn.setMinWidth(300);
+        descriptionColumn.setMaxWidth(Double.MAX_VALUE);
+
+        commandColumn.setSortable(false);
+        descriptionColumn.setSortable(false);
+
+        ObservableList<String[]> commandList = FXCollections.observableArrayList(
+                new String[] { "add", "Adds a contact to the list" },
+                new String[] { "clear", "Clears the contact list" },
+                new String[] { "delete", "Deletes contact(s) by index" },
+                new String[] { "remark", "Adds a remark to a contact" },
+                new String[] { "seed", "Seeds the contact list with sample data" },
+                new String[] { "edit", "Edits a contact in the list" },
+                new String[] { "list", "Lists all contacts" },
+                new String[] { "find", "Finds contact(s) by keyword" },
+                new String[] { "sort", "Sorts contacts by specific field" },
+                new String[] { "view", "Views more information regarding a contact" },
+                new String[] { "exit", "Exits the application" });
+
+        commandTable.setItems(commandList);
+        commandTable.setColumnResizePolicy(TableView.CONSTRAINED_RESIZE_POLICY_FLEX_LAST_COLUMN);
+
+        commandTable.setFixedCellSize(25);
+        commandTable.prefHeightProperty().bind(
+                Bindings.size(commandTable.getItems()).multiply(commandTable.getFixedCellSize()).add(30));
```

Comment:
Good job figuring this stuff out! Since JavaFX elements are generally less understood, perhaps some comments throughout method could be included to improve readability for future programmers?

Reply:
sure! will do